tests/app/service/test_rest.py

```diff
@@ -3385,3 +3390,122 @@ def test_get_returned_letter_summary(admin_request, sample_service):
     assert len(response) == 2
     assert response[0] == {'returned_letter_count': 2, 'reported_at': '2019-12-11'}
     assert response[1] == {'returned_letter_count': 1, 'reported_at': '2019-12-08'}
+
+
+@freeze_time('2019-12-11 13:30')
+def test_get_returned_letter(admin_request, sample_letter_template):
+    letter_1 = create_notification_history(template=sample_letter_template, client_reference='letter_1',
```

Comment:
What type of letter is letter_1?

So far I've got the following: 

letter_from_job: Created using a template made in our UI and sent using the UI using a CSV/spreadsheet
one_off_letter: Template created in the UI and sent as an individual through the UI
api_letter: Individual letter sent using the API using a template created in the UI
precompiled_letter: Sent as a PDF through the API
uploaded_letter: Uploaded through the UI 

On another note, I am absolutely loving the variable names you've given to these letters. It's some of the most readable code I've reviewed by you since joining I think :) Reading through this test has greatly improved my understanding of all the different types of letters so great work!